plugins/intrinsics/src/main/java/org/qbicc/plugin/intrinsics/core/CoreIntrinsics.java

```diff
@@ -1588,5 +1588,13 @@ private static void registerJdkInternalMiscUnsafeIntrinsics(final CompilationCon
         };
 
         intrinsics.registerIntrinsic(unsafeDesc, "isBigEndian0", emptyToBool, isBigEndian0);
+
+        InstanceIntrinsic unalignedAccess0 = (builder, instance, target, arguments) -> {
+            ClassContext c = builder.getCurrentElement().getEnclosingType().getContext();
+            LiteralFactory lf = c.getLiteralFactory();
+            return lf.literalOf(false);
```

Comment:
Isn't this platform dependent?  aarch64/x86/s390/ppc would all say true.  Others false.  

False is probably safe but may be pessimistic for some use cases

Reply:
#651